framework/common/test/encryption_test.cpp

```diff
@@ -61,27 +72,34 @@ int main()
           "keywrapkeys should be equal");
 
     {
-        uint8_t plaintext[1024];
+        uint8_t plaintext[50];
```

Comment:
It would be good to re-run this test for various plaintext sizes, so we actually test different amounts of padding. Ideally also include plaintext size < 16 bytes.

Reply:
Great idea, will do that.